### PR TITLE
#79 - test for PySlice

### LINKED ISSUE #79: Redirect to normalized project names support
According to [PEP 503](https://www.python.org/dev/peps/pep-0503/#specification) simple python repo:

> Repositories MAY redirect unnormalized URLs to the canonical normalized URL (e.g. /Foobar/ may redirect to /foobar/), however clients MUST NOT rely on this redirection and MUST request the normalized URL.

We should support this feature by introducing redirect slice. This slice should normalize last part of the request line and redirect to the new location.  
More details can be found in #59.

### PR DESCRIPTION
**Pull request by olenagerasimova**
Added test ensure that routing in `PySlice` works as expected and to close #79.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/artipie/pypi-adapter/issues/85#issuecomment-663166668))

@olenagerasimova Job `gh:artipie/pypi-adapter#85` is not assigned, can't get performer

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/artipie/pypi-adapter/issues/85#issuecomment-663166668))

@olenagerasimova Pull request [#85](https://github.com/artipie/pypi-adapter/issues/85) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

File: src/test/java/com/artipie/pypi/http/PySliceTest.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                    new Header("Content-Length", "145")
+                )
+            )
+        );
+    }
+
+    @Test
+    void redirectsToNormalizedPath() {
+        MatcherAssert.assertThat(
+            new PySlice(new InMemoryStorage()).response(
```
Comment (olegmoz) [inline src/test/java/com/artipie/pypi/http/PySliceTest.java:82]: @olenagerasimova `PySlice` with `InMemoryStorage` used in every test, have you considered initializing them once in setup method?

**Comment by olenagerasimova** [comment]
@olegmoz thanks, followed your suggestion, corrected regex and added one more test. Please, take a look

**Comment by 0crat** [comment]
Job `gh:artipie/pypi-adapter#85` is not assigned, can't get performer

**Pull request merged**